### PR TITLE
style[pokemonPage]: Adding and fixing styles in the pokemon page

### PR DESCRIPTION
**Pull request by jamoragar**
Challenge complete

### CODE REVIEW

File: css/css-basics_pokemon-page/css/styles.css [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
-/* your solution */
+* {
+  box-sizing: border-box;
+}
 
 /* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
 
 /* add selector */
- {
+body {
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:14]: If you put body here, the changes only affect elements inside the body. There is another solution to select all elements.

File: css/css-basics_pokemon-page/css/styles.css [hunk 2]
```diff
@@ -19,8 +21,9 @@
 /* ------- Headings Part 1 ------- */
 
 /* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
-
-/* your solution */
+h1 {
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:24]: Not wrong, but this way only h1 will be affected. What about other headings?

File: css/css-basics_pokemon-page/css/styles.css [hunk 3]
```diff
@@ -45,7 +51,15 @@
 */
 
 section {
-  /* your solution */
+  background-color: white;
+  margin: 10px;
+  padding: 20px;
+  border-style: solid;
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:57]: I think this puts a border on all all-around, but the task said _Add a solid 5px border with the color "orange" that is **only visible on the left. Hint: Use "border-left"**. :)_

File: css/css-basics_pokemon-page/css/styles.css [hunk 4]
```diff
@@ -45,7 +51,15 @@
...
+  border-color: orange;
+  border-width: 5px;
+}
+section:nth-of-type(2) {
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:61]: Did this work for you? Didn't for me, you might check again and maybe find a different solution if it didn't like in my case. (If it does - nice solution!)

File: css/css-basics_pokemon-page/css/styles.css [hunk 5]
```diff
@@ -59,22 +73,28 @@ section {
 */
 
 .subtitle {
-  /* your solution */
+  display: inline-block;
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:76]: I think, "inline-block" is not entirely correct.

File: css/css-basics_pokemon-page/css/styles.css [hunk 6]
```diff
@@ -59,22 +73,28 @@ section {
...
+  border-right: 3px solid gold;
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:77]: Not wrong, but there is an easier solution, so you only have one line to style the border left and right. :)

File: css/css-basics_pokemon-page/css/styles.css [hunk 7]
```diff
@@ -59,22 +73,28 @@ section {
...
+  border-left: 3px solid gold;
+  padding: 0 20px; /* Space between characters and border */
+}
+.favourite {
+  border-color: orange;
```
Comment (itsTHL) [inline css/css-basics_pokemon-page/css/styles.css:82]: Doesn't this put a border all around again? (Not sure)

**Review by itsTHL — CHANGES_REQUESTED** [review]
I left you some suggestions and comments, please check. :) 
Also, you seem to have worked on the buttons challenge in the same branch, as I can see that file, too. It's not supposed to be this way, I think. New branch for each challenge. :)

File: css/css-basics_pokemon-page/css/styles.css (hunk 1 above)
Comment (jamoragar) [reply css/css-basics_pokemon-page/css/styles.css:14]: you're right. I must use the `html` tag instead `body`

File: css/css-basics_pokemon-page/css/styles.css (hunk 4 above)
Comment (jamoragar) [reply css/css-basics_pokemon-page/css/styles.css:61]: actually yes... It's working for me.
I can show you if you want 😁

File: css/css-basics_pokemon-page/css/styles.css (hunk 1 above)
Comment (itsTHL) [reply css/css-basics_pokemon-page/css/styles.css:14]: Does that work? (Not what I was think of, haha. I actually thought the of the *)

**Pull request merged**